components/ActionSheet/src/Theming/MDCActionSheetController+MaterialTheming.m

```diff
@@ -51,6 +57,28 @@ - (void)applyThemeWithColorScheme:(id<MDCColorScheming>)colorScheme {
   UIColor *rippleColor = [colorScheme.onSurfaceColor colorWithAlphaComponent:kInkAlpha];
   self.inkColor = rippleColor;
   self.rippleColor = rippleColor;
+  self.traitCollectionDidChangeBlock = ^(MDCActionSheetController *_Nonnull actionSheet,
+                                         UITraitCollection *_Nullable previousTraitCollection) {
+    [actionSheet applyBackgroundColorToActionSheet:actionSheet withColorScheme:colorScheme];
```

Comment:
please check: `[self.traitCollection hasDifferentColorAppearanceComparedToTraitCollection:previousTraitCollection]` so to not re-apply the bgcolor if traitcollection changed for different reasons.

Reply:
Done